packages/labs/ssr-react/src/node/enable-lit-ssr.ts

```diff
@@ -20,9 +20,12 @@ import {wrapJsx, wrapJsxDev, wrapJsxs} from '../lib/node/wrap-jsx.js';
 /* eslint-disable @typescript-eslint/no-explicit-any */
 // The `.default ??` is used to get around inconsistent behavior with how
 // tools like webpack seem to do es module interop.
-Object.assign((React as any).default ?? React, {
-  createElement: wrapCreateElement(React.createElement),
-});
+if (React.createElement.name !== 'litPatchedCreateElement') {
```

Comment:
Couldn't `litPatchedCreateElement` be minified? I think it'd be safer to add a well-known property to the function in `wrap-create-element.ts` and check for that here. The property should be use computed property syntax to protect against minifiers and compilers.

Reply:
This is code meant to run in the server, though I suppose it's not impossible that server code could be minified.

Reply:
Since we have confirmation this fixes the issue for others, I'm happy to merge this as is. We can address it then if minification becomes an issue.